source/runner/Executor.ts

```diff
@@ -40,10 +40,10 @@ export class Executor {
    * @param {string} file the filepath to the Dangerfile
    * @returns {void} It's a promise, so a void promise
    */
-
   async runDanger(file: string, runtime: DangerfileRuntimeEnv) {
     const results = await runDangerfileEnvironment(file, runtime)
-    await this.handleResults(results)
+    const signature = githubSignature(runtime.context.danger.github.pr)
```

Comment:
Passing the `runtime` (or at least `runtime.context`) into `handleResults()` could be beneficial for danger/danger-js#99 too (see https://github.com/danger/danger-js/pull/99#discussion_r98305986).